### PR TITLE
Use paradox akka-theme artifacts from jcenter

### PR DESCRIPTION
**Pull request by raboof**
Let's see of artifactory sees them, too.

### CODE REVIEW

**Comment by raboof** [comment]
(after/when we merge this we should go through our jenkins jobs and make sure we use artifactory everywhere again)

**Review by patriknw — APPROVED** [review]
LGTM, merge if this has been verified

**Pull request merged**
